File: web-app/src/app/services/types/Email.ts
export type Email = {
  Id: number;
  To: string;
  Cc: string;
  Bcc: string;
  Subject: string;
  IsSent: boolean;
  Body: string;
  CreatedAt: string;
  SentAt: string;
};

export type PostEmail = {
  Id: number;
  To: string;
  Cc: string;
  Bcc: string;
  Subject: string;
  IsSent: boolean;
  Body: string;
  CreatedAt: string;
  SentAt: string;
};

File: Placements.DataAccess/Placement/Models/Email.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Email
{
    public long Id { get; set; }

    public string? To { get; set; }

    public string? Cc { get; set; }

    public string? Bcc { get; set; }

    public string? Subject { get; set; }

    public bool? IsSent { get; set; }

    public string? Body { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? SentAt { get; set; }

    public bool? HasAttachment { get; set; }

    public long? DocumentId { get; set; }

    public virtual Document? Document { get; set; }
}
